test/api_postRobotoff_test.dart

```diff
@@ -1,14 +1,15 @@
+
 import 'package:flutter_test/flutter_test.dart';
-import 'package:openfoodfacts/model/Insight.dart';
-import 'package:openfoodfacts/model/RobotoffQuestion.dart';
+//import 'package:openfoodfacts/model/Insight.dart';
```

Comment:
Similar comment regarding skipping tests https://pub.dev/packages/test#skipping-tests

Reply:
Thanks @peterwvj, yeah you have right I will use the skip annotation, it is better and cleaner